src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -501,7 +501,17 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
     * @return string. the Household account name
     */
     public static String strNameHHAccountForContact(Contact con) {
```

Comment:
Based on the requirement, if the Household account has two (or more) contacts, the Account should have the name from both contacts:

 **{!LastName} Household: Generates a Household name such as “Smith Household” or “Smith and Jones Household,” depending on the number of Contacts in the Household.

Also, in npsp, if the format is {!{!FirstName}} {!LastName} Household, then the result could be: 
Sam Smith Household
Sam and Sally Smith Household
Sam, Sally and Suzie Smith Household
Sam, Sally and Suzie Smith and Daphne Doe Household
Sam, Sally and Suzie Smith and Daphne and Donald Doe Household

Should we align with npsp implementation?